appserver/web/web-glue/src/main/java/com/sun/enterprise/web/pluggable/WebContainerFeatureFactory.java

```diff
@@ -54,7 +54,7 @@ public interface WebContainerFeatureFactory {
     
     public HealthChecker getHADBHealthChecker(WebContainer webContainer);
     
-    public ReplicationReceiver getReplicationReceiver(EmbeddedWebContainer embedded);
+    public ReplicationReceiver getReplicationReceiver(WebContainer webContainer);
```

Comment:
Is it necessary to keep this parameter? Its our own interface, and there's only one impl of it which ignores the parameter.